examples/super_resolution/main.py

```diff
@@ -145,4 +149,6 @@ def checkpoint():
     print("Checkpoint saved to {}".format(model_out_path))
 
 
+ProgressBar().attach(trainer)
```

Comment:
Let's remove `log_training_loss` handler and update `ProgressBar` options to display similar things.